helm/image-viewer/README.md

```diff
@@ -0,0 +1,3 @@
+# To get image-viewer to be discoverable to other microservices run the following command
+
+kc expose deployment local-image-viewer 
```

Comment:
add a service file to avoid this

Reply:
@matthewpeterkort  thanks for the review.  I actually have a service.yaml but this line is stopping it from being added?

https://github.com/ACED-IDP/gen3-helm/blame/ohsu-develop/.gitignore#L5

Reply:
I just used `git add -f` to add it

Reply:
Ok thank you. Yeah that seems odd. Might be good to remove that.